### PR TITLE
Add cost driver attributes

### PR DESCRIPTION
**Pull request by ldoran**
Add:
- costPerHour
- costPerHourForOvertime
- costPerKm
- fixedCost

Also, update requirements.txt to get `pip install -r requirements.txt` to work.

Design log: https://fieldaware.atlassian.net/wiki/spaces/SS/pages/199360615/Driver+model+updates+in+the+optimo+lib

### CODE REVIEW

File: optimo/models.py [hunk 1]
```diff
@@ -324,10 +324,20 @@ class Driver(BaseModel):
     :param work_shifts: ``list`` of :class:`optimo.models.WorkShift` objects
     :param skills: ``list`` of string skill ids for the driver
     :param speed_factor: ``numbers.Number`` denoting the driving speed adjustment
+    :param service_regions: ``list`` of
+        :class:`optimo.models.ServiceRegionPolygon` denoting the areas the driver
```
Comment (marzetas) [inline optimo/models.py]: Nitpick: leave a space after `:class:`

File: tests/test_models.py [hunk 2]
```diff
@@ -554,16 +554,61 @@ def test_service_regions(self, cls_name):
             .format(cls_name, 'ServiceRegionPolygon')
         assert err_msg == str(excinfo.value)
 
+    def test_cost_per_hour(self, cls_name):
```
Comment (marzetas) [inline tests/test_models.py]: I know this follows the pattern marked in the rest of the tests, but I think the name of these tests should be stating the fact that we are expecting a validation error (I was expecting a positive test case when I first read the test name).

File: optimo/models.py [hunk 3]
```diff
@@ -382,6 +396,18 @@ def validate(self):
...
                 )
 
+        if self.cost_per_hour is not None:
+            self.validate_type('cost_per_hour', Number)
+
+        if self.cost_per_hour_for_overtime is not None:
+            self.validate_type('cost_per_hour_for_overtime', Number)
+
+        if self.cost_per_km is not None:
+            self.validate_type('cost_per_km', Number)
```
Comment (adaszko) [inline optimo/models.py]: Shouldn't we be something keeping track of the type of units this is expressed in?  I'm guessing different businesses will want to express it in incomparable units.

Comment (ldoran) [reply optimo/models.py]: I think this should only be kilometers. The API user can do the conversion.

Comment (adaszko) [reply optimo/models.py]: I meant the 'cost' part, not the 'distance' part.

File: tests/test_models.py (hunk 2 above)
Comment (ldoran) [reply tests/test_models.py]: Suffixed "validation_fail" to each test

File: optimo/models.py (hunk 1 above)
Comment (ldoran) [reply optimo/models.py]: That's sphinx syntax. It won't render correctly otherwise.

File: optimo/models.py (hunk 3 above)
Comment (ldoran) [reply optimo/models.py]: Good point, I'll think about that

**Pull request merged**